detection_rules/rule.py

```diff
@@ -641,6 +650,7 @@ class ThresholdCardinality:
 
     type: Literal["threshold"]
     threshold: ThresholdMapping
+    alert_suppression: Optional[ThresholdAlertSuppression] = field(metadata=dict(metadata=dict(min_compat="8.12")))
```

Comment:
@vitaliidm Can you confirm this field is only available at 8.12? We can add a unit test or other validation once we do.

Reply:
yes, it is available at 8.12